tool/python/singa/layer.py

```diff
@@ -231,19 +231,23 @@ class Activation(Layer):
     def __init__(self, activation='stanh', topk=1):
         '''
         required
-          activation = (string)
+          activation = (string) // relu, sigmoid, tanh, stanh, etc.
         optional
-          topk       = (int)  // the number of results
+          topk       = (int)  // the number of results with k-highest accuracy
         '''
 
+        if activation != 'tanh':
+          print 'Warning: Tanh layer is not supported for CPU'
+
         self.name = activation
-        if activation == 'tanh': activation = 'stanh' # <-- better way to set?
         self.layer_type = enumLayerType(activation)
         super(Activation, self).__init__(name=generate_name(self.name),
                                          type=self.layer_type)
+
         if activation == 'softmaxloss':
             self.layer.softmaxloss_conf.topk = topk
```

Comment:
Activation function could be relu, sigmoid, tanh, stanh and softmax. But softmaxloss is a loss function. Hence it may confuse people if we use it as an activation function.